python/deploy.py

```diff
@@ -73,7 +73,7 @@ def run_bdist(deploy_dir):
 
     shutil.copytree(os.path.join(repo_dir, ".git"), ".git")
 
-    shutil.copy(os.path.join(script_dir, ".coveragerc"), "module/.coveragerc")
+    # shutil.copy(os.path.join(script_dir, ".coveragerc"), "module/.coveragerc")
```

Comment:
Did we work out if we need to add the converagerc back in?

Reply:
nope i think we dont need it but im not sure if i should leave it as it was when i released it

Reply:
Just delete this like, no since in maintaining cruft